apps/api/src/lib/algorand-adapter.ts

```diff
@@ -36,13 +42,14 @@ export default class AlgorandAdapter {
     )
 
     this.fundingAccount = algosdk.mnemonicToSecretKey(options.fundingMnemonic)
+    this.logger.info('Using funding account %s', this.fundingAccount.addr)
 
     this.testConnection()
   }
 
   async testConnection() {
     try {
-      const status = this.algod.status().do()
+      const status = await this.algod.status().do()
```

Comment:
🥇

Reply:
I did a major 🤦  for missing that one...